FHEM/90_SIGNALduino_un.pm

```diff
@@ -338,6 +360,25 @@ SIGNALduino_un_binflip($)
   return $flip;
 }
 
+#####################################
+sub SIGNALduino_un_b2h {
+    my $num   = shift;
+    my $WIDTH = 4;
+    my $index = length($num) - $WIDTH;
+    my $hex = '';
+    do {
+        my $width = $WIDTH;
+        if ($index < 0) {
```

Comment:
Es gibt ja bereits die Funktion sub SIGNALduino_b2h. Würde die es auch tun?

Reply:
Kann ich die Funktion SIGNALduino_b2h auch aus einem anderen Modul aufrufen?? Dachte die muss immer im Modul drin stehen.

Reply:
Ja, in FHEM kann man alle Funkionen aus allen geladenen Modulen aufrufen.

Reply:
Ich habe es angepasst und das war mit nicht bekannt, das dies funktioniert.
Nun sollten deine gewollten Änderungen gemodded sein mit dem https://github.com/RFD-FHEM/RFFHEM/pull/286/commits/2e5d46e993a6616f7db25a89aeee6a0e3c3bb969.

@sidey79, ready to takeover

PS:
>Hätte aber eine Idee, wie der Code etwas Überlichtlicher wird.
Wir können das aber auch getrennt optimieren .

Gern arbeite ich dir auch zu wenn du in Kurzform wie schon gemacht, dies mir zukommen lässt.